src/bindings/html/view.js

```diff
@@ -14,6 +14,8 @@ const observerConfig = {
   attributeFilter: ['data-l10n-id', 'data-l10n-args']
 };
 
+const readiness = new WeakMap();
```

Comment:
Ha, it could, if only it was better supported.  I'd like to steer clear of using `WeakSet` for now because of poor browser support.